index.html

```diff
@@ -247,48 +247,48 @@ <h3 class="table-header">Friday, October 29th</h3>
             <tr>
                 <td class="left">Check In</td>
                 <td class="middle">4th floor entrance</td>
-                <td class="right">5 PM</td>
+                <td class="right">5:00 PM</td>
             </tr>
             <tr>
                 <td class="left">Opening Ceremony</td>
                 <td class="middle">Dough 416</td>
-                <td class="right">6 PM</td>
+                <td class="right">6:00 PM</td>
             </tr>
 
             <tr>
                 <td class="left">Team Formation</td>
                 <td class="middle">Dough 416</td>
-                <td class="right">7 PM</td>
+                <td class="right">7:00 PM</td>
             </tr>
 
             <tr>
                 <td class="left">Dinner</td>
-                <td class="middle">TBA</td>
-                <td class="right">TBA</td>
+                <td class="middle">6th floor Commons</td>
+                <td class="right">7:30 PM</td>
             </tr>
 
             <tr>
                 <td class="left"><b>Hacking Commences</b></td>
                 <td class="middle">TBA</td>
-                <td class="right">TBA</td>
+                <td class="right">8:00 PM</td>
             </tr>
 
 
             <tr>
                 <td class="left">Sponsor Tech Talk</td>
-                <td class="middle">TBA</td>
-                <td class="right">TBA</td>
+                <td class="middle">Room 435</td>
+                <td class="right">8:30 PM</td>
             </tr>
             <tr>
                 <td class="left">Sponsor Tech Talk</td>
-                <td class="middle">TBA</td>
-                <td class="right">TBA</td>
+                <td class="middle">Room 435</td>
```

Comment:
Remove room numbers from tech talks for now as we still gotta confirm those 100%, as of now they are tentative but for the public let's keep TBA